packages/components/src/view/view.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * External dependencies
+ */
+import { BaseView } from '@wp-g2/styles';
+
+const View = BaseView;
```

Comment:
How is  this component different from the  "View" primitives we have? Should it be a wrapper around it, or does it replace it?

Reply:
`View` from primitives is a barebones (web) `div` or a `View` from React Native.

The `BaseView` from G2 is the entire foundation of the library's style system, shipped as a single base element:
https://github.com/ItsJonQ/g2/blob/master/packages/create-styles/src/create-style-system/create-core-element.js

> Should it be a wrapper around it, or does it replace it?

For the web implementation, IMO it should replace the `primitives/view` that we currently have (once we're ready).

---

If creating a `components/view` component is a concern at this stage, we can refactor things on the `@wp-g2/*` side so it would instead be:

```jsx
import { View } from '@wp-g2/styles';
```

Rather than:

```jsx
import { View } from '../view';
```

Let us know 🙌

Reply:
I don't know yet :) I think it's fine to keep it. but maybe for the  moment don't expose outside the components package especially since it has the css and as props which we might not want to expose.

Curious what other folks think though.

Reply:
I don't think we should ever expose `View` in the exports. I don't see any reason to expose it to the outside world, I can't think of the use case for it unless someone was building their own component library on top of G2, but then they should start with `create-styles` rather than `styles`/`View`.

There are other things too, `createComponent` shouldn't be exported either. If we don't want the "guts" of G2 to be exposed then we shouldn't export these things.